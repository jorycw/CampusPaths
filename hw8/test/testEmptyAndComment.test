# testing echoing empty and commented lines

# a one

# a two, a one


# a two
# three

# # #
q