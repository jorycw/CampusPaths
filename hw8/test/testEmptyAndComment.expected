Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): # testing echoing empty and commented lines

# a one

# a two, a one


# a two
# three

# # #